sql/moz-fx-data-shared-prod/accounts_backend_derived/monitoring_db_recovery_phones_counts_v1/query.sql

```diff
@@ -0,0 +1,12 @@
+SELECT
+  @as_of_date AS as_of_date,
+  SAFE.JSON_VALUE(lookupData, '$.countryCode') AS country_code,
+  COUNT(uid) AS total
+FROM
+  `moz-fx-data-shared-prod.accounts_db_external.fxa_recovery_phones_v1`
+WHERE
+  DATE(createdAt) = @as_of_date
+GROUP BY
+  country_code
+HAVING
+  country_code IS NOT NULL
```

Comment:
Are you sure you want to exclude rows that don't have a country code?  IMO knowing the proportion of rows which don't have a country code seems like it could be useful information.